src/Layer/babylon.highlightlayer.ts

```diff
@@ -879,6 +897,7 @@
             this._downSamplePostprocess.dispose();
             this._horizontalBlurPostprocess.dispose();
             this._verticalBlurPostprocess.dispose();
+            this._thresholdPostProcess.dispose();

```

Comment:
Need a if here as the threshold Pp can be undefined

Reply:
Done